config/local_variables.tfvars

```diff
@@ -0,0 +1,3 @@
+base_url    = "http://localhost:3000"
+client_id   = "terraform-client"
```

Comment:
Just questioning the naming of the client_id here, presume it corresponds to the React front end? If so, maybe something like 'callisto-client' might be more descriptive.

Reply:
Add also question it's need to be a variable. I feel that the client Id can be consistent across environments, it's the redirect URLs that make the client environment specific. i.e. this leads to less configuration changes in the deployed UI as well

Reply:
I've removed the two client variables, I have kept the base_url as this will not remain consistent across environments